api/client/samples/crop_models/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3618 @@
+{
```

Comment:
<p>The model failed for me here with this error:</p><p> </p><p><strong>NameError</strong>                                 Traceback (most recent call last)</p><p><strong>&lt;ipython-input-31-8a43cae6310d&gt;</strong> in &lt;module&gt;</p><p>      3 </p><p>      4 <strong># Generate a mask for the upper triangle</strong></p><p><strong>----&gt; 5 </strong>mask <strong>=</strong> np<strong>.</strong>zeros_like<strong>(</strong>corr<strong>,</strong> dtype<strong>=</strong>np<strong>.</strong>bool<strong>)</strong></p><p>      6 mask<strong>[</strong>np<strong>.</strong>triu_indices_from<strong>(</strong>mask<strong>)]</strong> <strong>=</strong> <strong>True</strong></p><p>      7 sns<strong>.</strong>set<strong>(</strong>font_scale<strong>=2)</strong></p><p> </p><p><strong>NameError</strong>: name 'np' is not defined</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>Fixed</p>